src/Processors/Formats/Impl/ParquetBlockOutputFormat.cpp

```diff
@@ -282,11 +281,15 @@ void ParquetBlockOutputFormat::writeRowGroup(std::vector<Chunk> chunks)
         writeUsingArrow(std::move(chunks));
     else
     {
-        Chunk concatenated = std::move(chunks[0]);
-        for (size_t i = 1; i < chunks.size(); ++i)
-            concatenated.append(chunks[i]);
-        chunks.clear();
-
+        Chunk concatenated;
```

Comment:
As I understand, this affects the ordering of the data. I am not sure this is ok... @vdimir @liuneng1994